sections/getting_started.tex

```diff
@@ -2,31 +2,24 @@ \section{Getting Started}
 
 \subsection{Installing Rust}
 
-\begin{frame}[c]{Rustup.rs}
-\begin{quote}
-«rustup is an installer for \\
-the systems programming language Rust» \\
-\vspace{0.5em}
-{\normalfont \small --- www.rustup.rs}
-\end{quote}
-\end{frame}
+\begin{frame}[fragile]{Rustup}
+	The best way to install and update the Rust toolchain is rustup
+	(\url{https://rustup.rs/}).
+
+	It makes it easy to:
+
+	\begin{itemize}
+		\item Install different Rust versions
+		\item Install cross compilation toolchains
+	\end{itemize}
+
+	\vspace{1em}
+
+	To download the installer script:
```

Comment:
We should probably mention that this is for Linux. For Windows there is an installer.

Reply:
It works on Windows too, if you have a sh-compatible shell.

Reply:
True, even rustup.rs mentions that you can use WSL on Windows.